src/accessibility/describe.js

```diff
@@ -379,23 +379,22 @@ function _descElementFallbackHTML(dummyDOM, cnvId, name, inner) {
           fallbackTableId}"><caption>Canvas elements and their descriptions</caption></table>`
       );
   }
-  //if it is the first time this element is being added to the table
-  if (dummyDOM.querySelector('#' + cnvId + fallbackTableId)) {
-    //create a table row for the element
-    let tableRow = document.createElement('tr');
-    tableRow.id = cnvId + fallbackTableElId + name;
-    dummyDOM.querySelector('#' + cnvId + fallbackTableId).appendChild(tableRow);
-    //update element description
-    dummyDOM.querySelector(
-      '#' + cnvId + fallbackTableElId + name
-    ).innerHTML = inner;
-    return dummyDOM;
-  }
+
+  //create a table row for the element
+  let tableRow = document.createElement('tr');
+  tableRow.id = cnvId + fallbackTableElId + name;
+  dummyDOM.querySelector('#' + cnvId + fallbackTableId).appendChild(tableRow);
+  //update element description
+  return [
+    dummyDOM.querySelector('#' + cnvId + fallbackTableElId + name),
```

Comment:
similar questions/comments here as for _describeFallbackHTML with this return clause and the insertion of the HTML earlier on